src/ImageSharp.Drawing/Shapes/PointOrientation.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Drawing
+{
+    /// <summary>
+    /// Represents the orientation of a point from a line.
+    /// </summary>
+    public enum PointOrientation
```

Comment:
Left public by mistake I assume.
```suggestion
    internal enum PointOrientation
```

Reply:
Yup!